src/main/webapp/app/shared/modal/AddMutationModal.tsx

```diff
@@ -1174,17 +1174,23 @@ const AddMutationInputOverlay = () => {
         <span style={{ fontStyle: 'italic', fontWeight: 'bold' }}>Add button</span> to annotate alteration(s).
       </div>
       <div className="mt-2">
-        <div>Examples:</div>
+        <div>Supported inputs:</div>
         <div>
           <ul style={{ marginBottom: 0 }}>
             <li>
-              Variant alleles seperated by slash - <span className="text-primary">R132C/H/G/S/L</span>
+              Variant alleles seperated by slash - <span className="text-success">R132C/H/G/S/L</span>
             </li>
             <li>
-              Comma seperated list of alterations - <span className="text-primary">V600E, V600K</span>
+              Comma seperated list of alterations - <span className="text-success">V600E, V600K</span>
             </li>
           </ul>
         </div>
+        <div className="mt-2">
+          For detailed list, refer to:{' '}
+          <Linkout to={'https://sop.oncokb.org/'}>
```

Comment:
We have `SopPageLink`

Reply:
Imported SopPageLink from public since we don't have in oncokb-curaiton